core/domain/draft_upgrade_services_test.py

```diff
@@ -142,9 +143,14 @@ def create_and_migrate_new_exploration(
         # The migration will automatically migrate the exploration to the latest
         # state schema version, so we set the latest schema version to be the
         # target_schema_version.
-        with self.swap(
-            feconf, 'CURRENT_STATE_SCHEMA_VERSION', int(target_schema_version)):
 
+        # Here we use object because create_and_migrate_new_exploration
+        # Is called by other functions, and using a decorator would
```

Comment:
These follow-on comment lines aren't supposed to be capitalized at the beginning.

Also I think that mocking a constant is a standard use case (and in some cases probably less invasive than mocking a function) so I don't think you need an explanation for it.